partcad/tests/unit/test_assembly_step.py

```
@@ -0,0 +1,339 @@
+    """
+    pkg = tmp_path / "pkg"
+    pkg.mkdir()
+    (pkg / "partcad.yaml").write_text("assemblies:\n  AeroAssembly:\n    type: step\n")
+    shutil.copy(AERO_STEP, pkg / "AeroAssembly.step")
+
+    ctx = pc.Context(str(pkg))
+    aero = ctx._get_assembly(":AeroAssembly")
+    assert aero is not None
+    try:
+        asyncio.run(aero.do_instantiate())
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)